smoke-tests/matrix/src/tomee-custom.dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG version
+ARG jdk
+ARG vm
+
+FROM tomee:${jdk}-jre-${version}-webprofile as default
+
+FROM adoptopenjdk:${jdk}-jdk-${vm}
+
+ENV SERVER_BASE=/usr/local/tomee
+COPY --from=default $SERVER_BASE $SERVER_BASE
+WORKDIR $SERVER_BASE
+
+COPY app.war $SERVER_BASE/webapps/
```

Comment:
Move COPY line to the end of the file. Then static EXPOSE and CMD layers can be cached.

Reply:
Done.